ruby.js

```diff
@@ -123,9 +123,12 @@ function balanceEvent(event) {
 }
 
 function completeCommand(args) {
+	let rctCompletePath = vscode.workspace.getConfiguration('ruby.rctComplete').commandPath;
+	args.push('--interpreter');
+	args.push(vscode.workspace.getConfiguration('ruby.interpreter').commandPath);
 	if (process.platform == 'win32')
```

Comment:
I suppose if we don't set a default value for this option, the value we get is an empty string, which makes this line of code not intuitive. Maybe pass in the default value?